netbox/ipam/api/serializers.py

```diff
@@ -352,15 +352,17 @@ class AvailableIPSerializer(serializers.Serializer):
     family = serializers.IntegerField(read_only=True)
     address = serializers.CharField(read_only=True)
     vrf = NestedVRFSerializer(read_only=True)
+    mask_length = serializers.IntegerField(read_only=True, allow_null=True)
```

Comment:
To be consistent with what has been done for /available-prefix/ endpoint:
don't change the `AvailableIPSerializer` serializer, but add a new one `IPMaskLengthSerializer` containing this `mask_length` variable (refer to what has been done for PrefixLengthSerializer).
I don't say it's the better solution, but it seems quite similar to the available-prefix endpoint, so I guess we must follow the same logic.

Reply:
done